packages/devextreme/js/__internal/ui/html_editor/utils/m_ai.ts

```diff
@@ -0,0 +1,109 @@
+import type {
+  HtmlEditorAIChangeStyleOption,
+  HtmlEditorAIChangeToneOption,
+  HtmlEditorAICommandName,
+  HtmlEditorAIToolbarItem,
+  HtmlEditorAITranslateOption,
+} from '@js/ui/html_editor';
+
+type DefaultOptionName =
```

Comment:
Why is it "default", why "name" if it's the whole option? 
```suggestion
type CommandOption =
```